src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        for (const key in action.extraData) {
+          state[key] = action.extraData[key];
+        };
+
+        break;
+
+      case 'removeProperties':
+        for (const key of action.keysToRemove) {
+          delete state[key];
+        }
+
+        break;
+
+      default:
+        // type === 'clear'
```

Comment:
mb better to add `case clear` explicitly here just in order for everyone to know that object can clear values

I strongly believe "default" behavior shouldn't clear the object cause it's too dangerous

you can define

```
default: {
  // do nothing
}
```

Reply:
👍